services/QuillLMS/app/views/pages/connect_tool.erb

```diff
@@ -47,8 +47,8 @@
       </div>
         <div class="walk-thru-step flex-row vertically-centered">
             <div class="content-wrapper flex-row space-between vertically-centered">
-                <img class="arrow arrow-1 lazyload" data-src=<%= "#{ENV['CDN_URL']}/images/tools/connect/arrow_1.png" %>>
-                <img class="negative-left-margin lazyload" data-src=<%= "#{ENV['CDN_URL']}/images/tools/connect/attempt_1.png" %> alt="connect image">
+                <img class="arrow arrow-1 lazyload" loading="lazy" src=<%= "#{ENV['CDN_URL']}/images/tools/connect/arrow_1.png" %>>
```

Comment:
So is the old `lazyload` stuff not working anymore?

Reply:
It's still working, I just opted to add [the newer property](https://web.dev/browser-level-image-lazy-loading/#improved-data-savings-and-distance-from-viewport-thresholds) since it has some more performant (it has pretty good browser support, but I left the old `lazyload` classes just in case)